doc/interop-test-descriptions.md

```diff
@@ -392,22 +392,24 @@ Asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### Metadata (TODO: fix name)
+### custom_metadata
 
-Status: Not yet implementable
+Status: Ready for implementation, beta requirement.
 
 This test verifies that custom metadata in either binary or ascii format can be
-sent in header and trailer.
+sent as initial-metadata by the client and as both initial- and trailing-metadata
+by the server.
 
 Server features:
 * [UnaryCall][]
+* [FullDuplexCall][]
 * [Compressable Payload][]
 * Ability to receive custom metadata from client in header and send custom data
-  back to client in both header and trailer. (TODO: this is not defined)
```

Comment:
This still needs to be defined. This should link to a server-side feature in the Server section.